scripts/02-filter-and-move.py

```diff
@@ -196,17 +203,26 @@ def call_back():
 
 
 if __name__ == '__main__':
-    start = time.time()
-    results = walk_in_parallel()
-
-    if not os.path.isdir(DIR_TO_CREATE):
-        os.makedirs(DIR_TO_CREATE)
-
     path_csv_out = str(Path(current_location, DIR_TO_CREATE, CSV_OUT))
     path_txt_out = str(Path(current_location, DIR_TO_CREATE, TXT_OUT))
-    df = pd.DataFrame(results, columns=['filename', 'class_type'])
-    df = df[df['class_type'] == 999]
-    df.to_csv(path_csv_out, index=False)
-    df['filename'].to_csv(path_txt_out, header=None, index=None)
-    end = time.time()
-    print('It took ' + str(end - start) + ' seconds')
+
+    if not args.split_only:
```

Comment:
Can you clarify the logic behind this `if`.
If there was no command line argument `--split-only`, then we do what?